mujoco_warp/testspeed.py

```diff
@@ -115,7 +116,7 @@ def _dataclass_memory(dataclass, total_bytes, conversion, unit):
   for field in fields(dataclass):
     value = getattr(dataclass, field.name)
     fieldinfo = []
-    if isinstance(value, mjw.Contact) or isinstance(value, mjw.Constraint):
+    if any(isinstance(value, mjw_type) for mjw_type in (mjw.Contact, mjw.Constraint, CCD)):
```

Comment:
`isinstance` accepts tuple as the second argument.
```suggestion
    if isinstance(value, (mjw.Contact, mjw.Constraint, CCD)):
```